src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/controllers/web/WorkflowRunControllerTests.kt

```diff
@@ -57,6 +57,27 @@ class WorkflowRunControllerTests
         assertThat(sut.getWorkflowRunSummaries()).isEqualTo(workflowRunSummaries)
     }
 
+    @Test
+    fun `can get workflow run summary`()
+    {
+        val requestBody = """{"ref": "18f6c5267c08bf017b521a21493771c6d3e774a5", "reports": [{"name": "missing"}]}"""
```

Comment:
```suggestion
        val requestBody = """{"ref": "18f6c5267c08bf017b521a21493771c6d3e774a5", "reports": [{"name": "missing"}]}""".trimIndent()
```

Reply:
@LekanAnni looks like we crossed wires and you reviewed an out of date PR. Do you still think the above changes are needed, namely, the trimIndents?

Reply:
It will make the test data solid and to avoid any form of fiddle, yes it is needed.

Reply:
@LekanAnni Why do you want to use `trimIndent` here? That just removes common indent from multi-line strings (https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/trim-indent.html), but this string is single line and doesn't have any indent so it won't make any difference